tests/test_gpu_treeshap.cu

```diff
@@ -173,34 +174,93 @@ TEST_P(ShapSumTest, ShapSum) {
   }
 }
 
+class ShapInteractionsSumTest : public ::testing::TestWithParam<
```

Comment:
That's new to me, thanks!

Reply:
Feels pretty clunky compared to hypothesis but it gets us more coverage at least.